building/tracks/new/prepare-for-launch.md

```diff
@@ -0,0 +1,56 @@
+# Prepare for launch
+
+Before your ready to launch your track, please ensure that the following tasks have been completed.
+
+## Request track icon
+
+In order to design an icon for the site, open a new issue in the [exercism/website-icons](https://github.com/exercism/website-icons/issues) repository, answering the following questions:
+
+- Does the language have an official logo? If so
+  - please include a link to an example
+  - please list the attribution rights of that logo
+- Does the language have an unofficial logo? If so
+  - please explain the ways in which people use the unofficial logo
+  - please include a link to an example
+  - please list the attribution rights of that logo
+
+## Add docs
+
+Add the following markdown documents:
+
+- `docs/ABOUT.md`: a short introduction to the language.
+- `docs/INSTALLATION.md`: describe what the student needs to install to allow working on the track on their local system using the CLI.
+- `docs/LEARNING.md`: links to learning resources.
+- `docs/RESOURCES.md`: links to useful resources.
+- `docs/TESTS.md`: describe everything related to running tests in the track.
+
+These documents will help the student get started with the track as well as learn about the track and its language.
+See the [docs documentation](/docs/building/tracks/docs) for more information.
+
+## Add exercise docs
+
+Add the following exercise documents:
+
+- `exercises/shared/.docs/help.md`: contains track-specific-wide instructions on how to get help
+- `exercises/shared/.docs/tests.md`: contains track-specific instructions on how to run the tests
+
+## Add code snippet
+
+Add a code snippet to the `docs/SNIPPET.txt` file.
+This snippet gives a visual impression of what the language syntax looks like.
+See the [docs documentation](/docs/building/tracks/docs) for more information.
+
```

Comment:
The code snippet instructions here seem to be a repeat of instructions earlier as well.